compass-geolocation-browser/src/commonMain/kotlin/dev/jordond/compass/geolocation/browser/internal/DefaultBrowserLocator.kt

```diff
@@ -40,6 +40,10 @@ internal class DefaultBrowserLocator : BrowserLocator {
     )
     override val locationUpdates: Flow<Location> = _locationUpdates
 
+    override suspend fun lastLocation(priority: Priority): Location? {
+        throw NotSupportedException()
+    }
```

Comment:
I wonder if we could try something like this instead of throwing outright.

```suggestion
    override suspend fun lastLocation(priority: Priority): Location? {
        try {
            return current(priority)
        }  catch (error: NotFoundException) {
            return null
        }
    }
```

Return `null` if the location is not found, and throw on any other exception. It won't be as immediate as the Android or iOS platforms, but I think its better than throwing a `NotSupportedException`